### DISCUSSION TITLE
Update to 1.0.9 in plans?

### DISCUSSION THREAD

**Discussion by everonline** [saucecontrol/BrotliIIS#8] [General]
First great work here, only 29 days ago Microsoft solved the memory leak on their br version, this one works flawlessly.

That being said any idea to bring this release up to version 1.0.9?

Thanks again.

**Comment by saucecontrol** [comment]
Funny you should bring that up.  I'm just setting up a bunch of new servers this week and was debating whether to keep using my own plugin or switch over to the MS one.  I guess I'll keep using mine :)

I've just done a new build and will upload it after I finish testing.

**Reply by everonline** [reply]
That is great! Your own plugin its really straight forward to install and flawless in terms of functionality, we were keen to go with M$ one but is normally over bloated and as mention only recently they solved a memory leak, the fact they also have some sort of GZIP reimplementation don't please us.

Thanks again for your work.
